src/visning/Visning.tsx

```diff
@@ -24,28 +28,33 @@ const Visning: FunctionComponent<Props> = ({ fnr }) => {
     }, [fnr]);
 
     if (kandidat.status !== Status.Suksess) {
-        return null;
+        return <div></div>;
     }
 
     return (
-        <>
-            <button onClick={navigerTilRegistreringsside}>registrer</button>
-            <Behovgruppe
-                overskrift="Fysisk tilrettelegging"
-                beskrivelse="Behov for fysisk tilrettelegging på arbeidsplassen"
-                behov={kandidat.data.fysiskeBehov}
-            />
-            <Behovgruppe
-                overskrift="Arbeidshverdagen"
-                beskrivelse="Behov for tilpasninger i arbeidshverdagen"
-                behov={kandidat.data.arbeidsmiljøBehov}
-            />
-            <Behovgruppe
-                overskrift="Utfordringer med norsk"
-                beskrivelse="Kandidaten har utfordringer med å:"
-                behov={kandidat.data.grunnleggendeBehov}
-            />
-        </>
+        <div className="Visning">
+            <div className="sistendret">
+                <Normaltekst>Sist endret: {kandidat.data.sistEndret}</Normaltekst>
+            </div>
+            <div className="Behovkategorier">
+                <Behovgruppe
+                    overskrift="Fysisk tilrettelegging"
```

Comment:
Har vi definert disse overskriftene i domenemodellen? Hvis ikke, bør vi det?

Reply:
Vi kan ta det samtidig med vi legger inn den siste kategorien i ny pr.

> Er filene formattert med prettier? :D

Jeg manglet det for less filer, men har fikset det nå.